solutions/standard-plus-vsi/variables.tf

```diff
@@ -255,6 +255,28 @@ variable "existing_monitoring_instance_crn" {
   default     = null
 }
 
+#################################################
+# Optional Parameters SCC Workload Protection
+#################################################
+
+variable "enable_scc_wp" {
+  description = "Set to true to enable SCC Workload Protection and install and configure the Sysdig agent on all VSIs and PowerVS instances in this deployment."
+  type        = bool
+  default     = false
+}
+
+variable "ansible_vault_password" {
+  description = "Vault password to encrypt ansible variable file for SAP installation."
+  type        = string
+  sensitive   = true
+  default     = null
+
+  validation {
```

Comment:
Move this validation to root module landing zone. Then this will be not be required to mention in other solution as well